README.md

```diff
@@ -1,8 +1,31 @@
 # Redux-cycles
+
+<div align="center">
+  <img src="logo.png" alt="Cycle.js + Redux = <3" />
```

Comment:
I think will more appropriate to have redux + cycle = love (in the same order as the name 'redux-cycles') as it is cycle in redux not vice-versa I would argue. Otherwise love this :) What if we simply pull logo from both project to make this (just an idea) ?